.circleci/config.yml

```diff
@@ -76,9 +76,8 @@ jobs:
       - run:
           name: Test Production Stack
           command: |
+            pip install j2cli==0.3.10
```

Comment:
I don't think this is necessary since you added this to the requirements file.

Reply:
@levlaz it was failing without it I'm pretty sure. At this point the requirements.yaml does not come into play because the container is part of the docker-compose configuration.

If anything I think I should remove it from the requirements.yaml and test

Reply:
Could you verify that it doesn't work when it's in the `requirements.txt`? It'd be better to keep all our module dependencies and versions in one place if it's possible.